cmd/commands/cli/command.go

```diff
@@ -159,20 +161,107 @@ func (c *cliApp) handleActions(line string) {
 	}
 }
 
+func (c *cliApp) service(argsString string) {
+	var action, providerID, serviceType, id string
+	var options json.RawMessage
+
+	help := func() {
+		fmt.Println("service <action> [args]")
+		fmt.Println("	start	<ProviderID> <ServiceType> [options]")
+		fmt.Println("	stop	<ServiceID>")
+		fmt.Println("	list")
+		fmt.Println("	get	<ServiceID>")
+		fmt.Println("")
+		fmt.Println(`example: service start 0x7d5ee3557775aed0b85d691b036769c17349db23 openvpn {"port":1190, "protocol":"UDP"}`)
+	}
+
+	args := strings.Fields(argsString)
+	if len(args) == 0 {
+		help()
+		return
+	}
+
+	action = args[0]
+	switch action {
+	case "start":
```

Comment:
could we extract all the cases to separate functions? we'd have smaller blocks.

Reply:
Extracted.